### ISSUE TITLE
Custom line styles

### ISSUE THREAD

**Issue by edemaine** [edemaine/cocreate#61]
Similar to colors: have some fixed styles and then allow custom creation (like #14).

This will be a challenge to implement for pen strokes

**Comment by mehtank** [comment]
There may be other relevant svg presentation attributes to include; e.g. a highlighter can have an [opacity](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/opacity) (though it would be nice to make it wider too); you can create arrows with [marker-start](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/marker-start) and [marker-end](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/marker-end), etc.

**Comment by ndattani** [comment]
In addition to dashed and dotted lines, I'd very much like to be able to do lines _**with arrows**_ because I draw a lot of vectors! While at it, I would suggest being able to "fill" a rectangle or circle, just like we can in PowerPoint and other presentation-making software.

**Comment by edemaine** [comment]
> In addition to dashed and dotted lines, I'd very much like to be able to do lines _**with arrows**_ because I draw a lot of vectors!

Agreed. This is currently in issue #35. It's a reasonably high priority for me too.

> While at it, I would suggest being able to "fill" a rectangle or circle, just like we can in PowerPoint and other presentation-making software.

This already exists: if you shift-click on a color, it sets the fill.  See [the user guide](https://github.com/edemaine/cocreate/blob/main/doc/README.md) for more details.

**Comment by edemaine** [comment]
FYI, both arrows and line styles are now available. (Line styles will deploy shortly.)

**Comment by ndattani** [comment]
Very nice! Also, I hadn’t noticed that I could make rectangles be filled by clicking on the raindrop button, but it works well for me now :)
